en/flight_controller/airlink.md

```diff
@@ -0,0 +1,1003 @@
+# Sky-Drones AIRLink
+
+:::warning
+PX4 does not manufacture this (or any) autopilot.
+Contact the [manufacturer](https://sky-drones.com/) for hardware support or compliance issues.
+:::
+
+[AIRLink](https://sky-drones.com/airlink>) stands for Artificial Intelligence & Remote Link. The unit includes a cutting-edge drone autopilot, AI mission computer and LTE/5Gs connectivity unit. AIRLink helps to reduce the time to market for new drone manufacturers from years and months down to weeks.
+
+![AIRLink](../../assets/flight_controller/airlink/airlink-main.jpg)
+
+:::note
+This flight controller is [manufacturer supported](../flight_controller/autopilot_manufacturer_supported.md).
+:::
+
+AIRLink has two computers and integrated LTE Module: 
+
+   - The flight control computer (autopilot) has a triple-redundant vibration-dampened and temperature-stabilized IMU. 
+   - The powerful AI mission computer enables advanced drone software features like computer vision and obstacle avoidance, digital HD video streaming, and payload data streaming.
+   - LTE/5G and WiFi connectivity modules provide permanent broadband internet connection which is enabler for remote workflows.
+
+
+![Three in one](../../assets/flight_controller/airlink/airlink-3in1.png)
```

Comment:
I would remove this image, or move it. Fine on a marketing doc, but just a waste of information space here - says nothing you didn't have in your list above.

Reply:
Yes, makes sense, removed now.